web/src/features/hfiCalculator/pages/HfiCalculatorPage.tsx

```diff
@@ -33,11 +35,24 @@ import { PST_UTC_OFFSET, PST_ISO_TIMEZONE } from 'utils/constants'
 import PrepDaysDropdown from 'features/hfiCalculator/components/PrepDaysDropdown'
 import { FireCentre } from 'api/hfiCalcAPI'
 import { HFIPageSubHeader } from 'features/hfiCalculator/components/HFIPageSubHeader'
-import { cloneDeep, isNull, isUndefined } from 'lodash'
+import { isNull, isUndefined } from 'lodash'
 import HFIErrorAlert from 'features/hfiCalculator/components/HFIErrorAlert'
 import DownloadPDFButton from 'features/hfiCalculator/components/DownloadPDFButton'
 import EmptyFireCentreRow from 'features/hfiCalculator/components/EmptyFireCentre'
 
+function constructPlanningAreaFireStarts(
```

Comment:
Shouldn't this happen at the API boundary too? In `web/src/api/hfiCalculatorAPI.ts` or in the fetch functions

Reply:
This function should be thrown away when fetchHFIResult and fetchPDFDownload get thrown away, so not worried about it right now. Will add a comment to that effect though.